packages/js/components/src/utils.tsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { isValidElement, Fragment } from 'react';
+import { Slot, Fill } from '@wordpress/components';
+import { cloneElement, createElement } from '@wordpress/element';
+
+// TODO: Right now the build is breaking if I don't reference createElement in some way.
+// console.debug( 'createElement', typeof createElement );
```

Comment:
I'm currently having an issue where an error is thrown on the page if `createElement` isn't referenced in the file. I have to uncomment this to avoid it; is this just me?